app/assemble/partials/300-javascript/310-writing-and-formatting-javascript.hbs

```diff
@@ -26,9 +26,10 @@ Formatting the language statements and patterns should follow these basics:
 * Conditional statements go on a new line followed by the opening brace.
 * Else/else go on the same line as the brace.
 * Use type strict checks with === as opposed to == whenever possible.
+* Use += and -= instead of ++ and --.
 {{/markdown}}
 {{#code "javascript"}}
-for (var i = 0, len = arr.length; i < len; i++) {
+for (var i = 0, len = arr.length; i < len; i += 1) {
```

Comment:
Here are two posts on Stackoverflow that interprets this scenario...

http://stackoverflow.com/questions/8452317/do-loops-check-the-array-length-every-time-when-comparing-i-against-array-length

http://stackoverflow.com/questions/17989270/for-loop-performance-storing-array-length-in-a-variable